webui/common/src/components/FileView/FileView.tsx

```diff
@@ -90,7 +81,7 @@ export class FileView extends React.PureComponent<IFileViewProps> {
             </ButtonGroup>
           </FormGroup>
           <FormGroup className="col-2">
-            <Button tag={Link} to={`${queryStringPrefix}?path=${path}${queryStringSuffix}`} color="secondary">Go</Button>
+            <Button tag={Link} to={`${queryStringPrefix}?path=${encodeURIComponent(path || '')}${queryStringSuffix}`} color="secondary">Go</Button>
```

Comment:
Was this previously a bug?

Reply:
the path is always decoded now and we encode it when it's used to navigate, so I added that to be consistent